plugins/PRAplugin/doc/include/MarkovCategorical.tex

```diff
@@ -0,0 +1,69 @@
+\section{Markov Categorical Distribution}
```

Comment:
this is not in the plagin....but in the RAVEN distrubution... you should move this in there...

Actually it is the same for the other PostProcessors.... are in RAVEN for now, so it should be part of the Plugin documenation